js/id/behavior/hash.js

```diff
@@ -36,6 +36,11 @@ iD.behavior.Hash = function(context) {
                 '/' + center[0].toFixed(precision) +
                 '/' + center[1].toFixed(precision);
 
+        if (context.indoor().enabled())
+            q.level = context.indoor().level();
+        else
+            delete q.level;
```

Comment:
@zbycz Please never ever omit curly braces around if-else, it’s generally considered as a bad practice and there are many documented cases when it caused a security issue.